### PR TITLE
updates the how it works & faq links to point to the staging site

### LINKED ISSUE #233: Integrate Primary Navigation external links into Sidebar

### LINKED ISSUE #234: "How it works" cross-link from Welcome Page
URL: tbd
Behavior: open in new tab

### LINKED ISSUE #284: replace links in core add-on with staging site links

### PR DESCRIPTION
**Pull request by hamilton**
Closes #284, closes #233, closes #234

Checklist for reviewer:

- [x] The description should reference a bug or github issue, if relevant.
- [x] There must be a [`CHANGELOG.md`](./CHANGELOG.md) entry for any non-test change.
- [ ] Any change to the NPM commands must be carefully reviewed to make sure it won't break the Add-ons pipeline.
- [ ] Any version increase must follow the [release process](./docs/RELEASE_PROCESS.md).

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -51,6 +51,8 @@ export default {
     replace({
       __STORE_IMPLEMENTATION__: JSON.stringify(STORE_MOCK),
       __API_ENDPOINT__: production ? "web-extension" : "web",
+      __HOW_IT_WORKS_URL__: "https://rally-stage.bespoke.nonprod.dataops.mozgcp.net/how-rally-works/",
+      __FAQ_URL__: "https://rally-stage.bespoke.nonprod.dataops.mozgcp.net/how-rally-works/faqs/"
```
Comment (Dexterp37) [inline rollup.config.js:55]: nit: would it make more sense to just specify the base URL once and build the full URI with that? e.g. `__BASE__: "https:.../"` and then `__FAQ_URL__: __BASE__ + "how-rally-works/"`?

Comment (hamilton) [reply rollup.config.js:55]: yep, will do that

File: src/routes/Sidebar.svelte [hunk 2]
```diff
@@ -106,7 +106,7 @@ li a {
   <ul slot="navigation" in:fly={{duration: 800, delay: 300, x: -10}}>
     <li><button class:active={currentView === 'current-studies'} on:click={send('current-studies')}>Current Studies</button></li>
     <li>
-      <a class="external" href="/studies">FAQ
+      <a class="external" href="__BASE_SITE__/__FAQ_PATH__">FAQ
```
Comment (Dexterp37) [inline src/routes/Sidebar.svelte:109]:
```suggestion
      <a class="external" href="__BASE_SITE__/how-rally-works/faqs/">FAQ
```

nit: this is used exactly once in the code. Is there any advantage in making it a constant vs hardcoding it?

File: src/routes/welcome/Content.svelte [hunk 3]
```diff
@@ -120,7 +120,7 @@
         Get Started
       </Button>
       <div class="how-it-works">
-        <a href="example.com">how it works
+        <a href="__BASE_SITE__/__HOW_IT_WORKS_PATH__">how it works
```
Comment (Dexterp37) [inline src/routes/welcome/Content.svelte:123]:
```suggestion
        <a href="__BASE_SITE__/how-rally-works/">how it works
```

File: rollup.config.js [hunk 4]
```diff
@@ -51,6 +51,11 @@ export default {
     replace({
       __STORE_IMPLEMENTATION__: JSON.stringify(STORE_MOCK),
       __API_ENDPOINT__: production ? "web-extension" : "web",
+      // the following replacements build the site URLs.
+      // In the templates, use (for example) __BASE_SITE_URL__/__FAQ_PATH__
+      __BASE_SITE__: "https://rally-stage.bespoke.nonprod.dataops.mozgcp.net",
+      __HOW_IT_WORKS_PATH__: "how-rally-works/",
+      __FAQ_PATH__: "how-rally-works/faqs/"
```
Comment (Dexterp37) [inline rollup.config.js:58]:
```suggestion
```

These are used exactly once in the code: is there any advantage in keeping them in the config file?

**Review by Dexterp37 — APPROVED** [review]
I have a few nits below (I recommended the changes so that you can apply them, if you want, from the GitHub UI): they are not mandatory, so apply them at your discretion.

**Pull request merged**